tests/ui/llm_studio_page.py

```diff
@@ -0,0 +1,242 @@
+
+    def import_dataset_from_azure(
+        self, connection: str, container: str, dataset_name: str
+    ):
+        self.import_dataset("Azure Blob Storage")
+        self.get_by_test_id(self.AZURE_CONN_STRING).fill(connection)
+        self.get_by_test_id(self.AZURE_CONTAINER).fill(container)
+        self.get_by_test_id(self.AZURE_FILENAME).fill(dataset_name)
+        self.continue_button().click()
+
+    def import_dataset_from_kaggle(
+        self, kaggle_command: str, username: str, secret: str
+    ):
+        self.import_dataset("Kaggle")
+        self.get_by_test_id(self.KAGGLE_COMMAND).fill(kaggle_command)
+        self.get_by_test_id(self.KAGGLE_USERNAME).fill(username)
+        self.get_by_test_id(self.KAGGLE_SECRET_KEY).fill(secret)
+        self.continue_button().click()
+
+    def delete_dataset(self, dataset_name: str):
+        # Go to dataset page
+        self.view_datasets()
+        self.get_by_test_id(self.DATASET_LIST_DELETE_SELECTOR).click()
+        # Locate dataset to delete
+        self.page.get_by_role("gridcell", name=dataset_name).click()
+        # Confirm dataset deletion
+        self.get_by_test_id(self.DATASET_DELETE_DIALOG_SELECTOR).click()
+        # Delete dataset
+        self.get_by_test_id(self.DATASET_DELETE_SELECTOR).click()
+
+    def view_datasets(self):
+        locator = self.page.get_by_role("button", name="View datasets")
+        locator.click()
+
+    def assert_dataset_deletion(self, dataset_name: str):
+        self.view_datasets()
+        dataset = self.page.get_by_role("button", name=dataset_name)
+        # Assert that the element not found
+        expect(dataset).not_to_be_visible()
+
+    def create_experiment(self, name: str):
+        # Get locator element
+        locator = self.page.get_by_role("button", name="Create experiment")
+        locator.click()
+        self.experiment_name(name)
+
+    def slider(self, slider_selector, target_value: str):
+        is_completed = False
+        i = 0
+        # Get the slider element
+        slider = self.get_by_test_id(slider_selector)
+        slider.click()
+        # Get the bounding box of the slider
+        bounding_box = slider.bounding_box()
+        x1 = bounding_box["x"]
+        y = bounding_box["y"] + bounding_box["height"] / 2
+
+        while not is_completed:
+            self.page.mouse.move(x1, y)
+            self.page.mouse.down()
+            x2 = bounding_box["x"] + bounding_box["width"] * float(i) / 100
+            self.page.mouse.move(x2, y)
+            self.page.mouse.up()
+            value_now = slider.get_attribute("aria-valuenow")
+
+            if value_now == target_value:
+                is_completed = True
+            else:
+                # Move the slider a little bit (adjust the step as needed)
+                step = 0.1  # Adjust this value based on your requirements
+                x1 = x2
+                i += step
+
+    def run_experiment(self):
+        self.page.wait_for_timeout(3000)
```

Comment:
1. let's use a constant with a meaningful name, if we need need a shorter timeout than the default one
2. why do we need a shorter timeout? for a quick test failure?

Reply:
I was trying to implement sleep here @mnezh. This UI was crashing randomly at this point while tests locally

Reply:
Let me find a better way to do this

Reply:
@itsmunishbhardwaj if the UI is crashing, probably it needs some bugfix? Could you please investigate the bug and raise a ticket for @maxjeblick  to review?

Reply:
Its a random event @mnezh Im unable to capture it. Theres a "sorry for inconvenience" message that appears during a crash